mkt/extensions/migrations/0003_populate_extension_uuid.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def add_uuid_to_exising_extensions(apps, schema_editor):
+    Extension = apps.get_model('extensions', 'Extension')
+    extensions = Extension.objects.all()
+    for extension in extensions:
+        extension.uuid = extension._meta.get_field('uuid')._create_uuid()
```

Comment:
Yeah. It's not that secret, really all it does is a `uuid.uuid4()` call. I could replace it - I wanted to use what the field really uses behind the scenes